src/universal-dapp.js

```diff
@@ -192,7 +192,6 @@ UniversalDApp.prototype.getInstanceInterface = function (contract, address, $tar
                 return;
 
             $event = $('<div class="event" />');
-
```

Comment:
Every line that @axic changed in app.js was extracted to the new modules. In the process, I made the same formatting updates. (Except for jsonInterface vs interface, which is about to be committed.)